pkg/containerimagebuilderpusher/docker.go

```diff
@@ -92,13 +92,11 @@ func (d *Docker) TransformOnbuildArtifactPaths(onbuildArtifacts []runtime.Artifa
 }
 
 func (d *Docker) GetBaseImageRegistry(registry string) string {
-
-	// keep base image registry as is
-	return ""
+	return d.builderConfiguration.DefaultBaseRegistryURL
```

Comment:
Should be always "" for docker imo, but let's do the full scenario matrix offline

Reply:
all good